x-pack/plugin/core/src/main/resources/metrics-default-mappings.json

```diff
@@ -0,0 +1,57 @@
+{
+  "template": {
+    "mappings": {
+      "dynamic_templates": [
+        {
+          "strings_as_keyword": {
+            "mapping": {
+              "ignore_above": 1024,
+              "type": "keyword"
+            },
+            "match_mapping_type": "string"
+          }
+        }
+      ],
+      "date_detection": false,
+      "properties": {
```

Comment:
@exekias @roncohen I was wondering if we should add anything related to detecting numbers to our default template?

Reply:
uhm good question, it's my understating that beats already send metrics correctly encoded as numbers in the JSON request. That would make Elasticsearch use float and integer. Probably this is fine as a default, WDYT?

Reply:
Beats is sending `1.0` if we set it as float instead of `1`? I must confess, I don't remember. But if that is the case, ++ on having even number detection on. 

@dakrone What are all the option we have here?

Reply:
@ruflin all `numeric_detection` does is to make strings that are numbers (ie `"1.3"` instead of `1.3`) be numbers rather than strings. If beats is already sending data in the correct format (numbers instead of strings), I don't think it would buy us anything other than requiring the string to be read on every new field introduced.

Reply:
++, sounds like we can ignore it then for now.